aws/ec2/deploy/home/ec2-user/snappydata/zeppelin-setup.sh

```diff
@@ -159,13 +162,13 @@ if [[ -e "${ZEP_DIR}/conf/interpreter.json" ]]; then
     echo "${LEADS}" > lead_list
     LEAD_HOST=`cat lead_list | sed -n '1p'`
   fi
-  sed -i "/group\": \"snappydata\"/,/isExistingProcess\": false/{s/isExistingProcess\": false/isExistingProcess\": snappydatainc_marker,/}" "${ZEP_DIR}/conf/interpreter.json"
-  sed -i "/snappydatainc_marker/a \"host\": \"${LEAD_HOST}\",\n\
-         \"port\": \"${LEAD_PORT}\"" "${ZEP_DIR}/conf/interpreter.json"
+  sed -i "/group\": \"snappydata\"/,/isExistingProcess\": false/{s/isExistingProcess\": false/isExistingProcess\": snappydatainc_marker/}" "${ZEP_DIR}/conf/interpreter.json"
+  sed -i "/snappydatainc_marker/a \"host\": \"localhost\",\n \"port\": \"3768\"," "${ZEP_DIR}/conf/interpreter.json"
```

Comment:
Why are we hardcoding the host to be localhost? If the zeppelin is running in its own instance, it'll fail.